inc/com/centreon/engine/notification.hh

```diff
@@ -59,6 +59,8 @@ class notification {
   notifier::reason_type get_reason() const;
   uint32_t get_notification_interval() const;
   bool sent_to(const std::string& user) const;
+  void insert_contacts(std::set<std::string> contact_notified);
+  std::set<std::string> get_contacts();
```

Comment:
If you don't want to make a copy of contacts and just want to use this list, it is better to declare something like:
const std::set<std::string>& get_contacts() const;